### PR TITLE
Ilkhom.a.geek 4.landing page

### PR DESCRIPTION
**Pull request by ilhom2497**
added react-router, created LandingPage component, and added Header and Links components. Changed sizing to rem

https://geeks-batch.atlassian.net/browse/GEEK-4

### CODE REVIEW

File: src/LandingPage/LandingPage.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Poppins:wght@300;400;500;600&display=swap');
```
Comment (zheksha) [inline src/LandingPage/LandingPage.css:1]: Put it under `index.css` so other components CSS has access to the same font

File: src/LandingPage/components/Links.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      <div className="row text-center">
+        <div class="col-6 ps-5">
+          <div>
+            <img src={companies} alt="" />
+          </div>
+          <div className="mt-3">
+            <Link>Companies</Link>
+          </div>
+        </div>
+        <div class="col-6 pe-5">
```
Comment (zheksha) [inline src/LandingPage/components/Links.jsx:22]: rename to `className`

File: src/LandingPage/components/Links.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import trucks from '../images/trucks.jpeg'
+import mechanics from '../images/mechanics.jpeg'
+import trafficLawyers from '../images/traffic-lawyers.jpeg'
+import other from '../images/other.jpeg'
+
+const Links = () => {
+  return (
+    <div className="col-6 pe-5 ps-5">
+      <div className="row text-center">
+        <div class="col-6 ps-5">
```
Comment (zheksha) [inline src/LandingPage/components/Links.jsx:14]: rename to `className`

**Review by zheksha — APPROVED** [review]

File: src/LandingPage/LandingPage.css (hunk 1 above)
Comment (ilhom2497) [reply src/LandingPage/LandingPage.css:1]: done

File: src/LandingPage/components/Links.jsx (hunk 2 above)
Comment (ilhom2497) [reply src/LandingPage/components/Links.jsx:22]: done

**Pull request merged**